index.html

```diff
@@ -18,274 +18,317 @@
-                        </div>
-                        <ul id="list-of-languages">
-                            <li>
-                                <div class="skill-box row-container">
-                                    <div id="skill-icon-1" class="skill-icon"></div>
-                                    <p>JavaScript</p>
-                                </div>
-                            </li>
-                            <li>
-                                <div class="skill-box row-container">
-                                    <div id="skill-icon-2" class="skill-icon"></div>
-                                    <p>HTML</p>
-                                </div>
-                            </li>
-                            <li>
-                                <div class="skill-box row-container">
-                                    <div id="skill-icon-3" class="skill-icon"></div>
-                                    <p>CSS</p>
-                                </div>
-                            </li>
-                        </ul>
-                    </div>
-                </li>
-                <li>
-                    <div id="frameworks" class="skill-header row-container">
-                        <p>Frameworks</p>
-                        <i class="fa-solid fa-chevron-right"></i>
-                    </div>
-                </li>
-                <li>
-                    <div id="skills" class="skill-header row-container">
-                        <p>Skills</p>
-                        <i class="fa-solid fa-chevron-right"></i>
-                    </div>
-                </li>
-            </ul>
-        </div>
-    </section>
-
-    <section id="contact-form">
-        <div class="column">
-            <div>
-                <h2>Contact me</h2>
-                <p>If you have an application you are interested in developing, a feature that you need built or a
-                    project that needs coding.
-                    I’d love to help with it!</p>
+                </form>
             </div>
-            <form action="https://formspree.io/f/xyyaojgk" method="post">
-                <ul>
-                    <li>
-                        <input type="text" class="form-input" name="name" id="name" placeholder="Full Name" maxlength="30" required />
-                    </li>
-                    <li>
-                        <input type="email" class="form-input" name="email" id="email" placeholder="Email address" required />
-                    </li>
-                    <li>
-                        <textarea placeholder="Hey, I'd like to connect to..." maxlength="500" name="message" rows="10" cols="30" required></textarea>
-                    </li>
-                    <li>
-                        <input type="submit" class="contact-btn" value="Get in touch" />
-                    </li>
-                </ul>
-            </form>
-        </div>
-    </section>
+        </section>
+    </div>
 
 </body>
```

Comment:
- [ ] You have done well, and your hard work is paying off. I think there's just very little to work on here. Kindly access your webpage and see how the page is overflowing on bigger screens (`>= 768px`) and how the width on smaller screens is not covering the available space, leaving unexpected white spaces on both sides of the content.

|Desktop Design| Smaller screens|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/236651885-0f531319-100c-486e-b6f8-300ce44813c2.png)|![image](https://user-images.githubusercontent.com/99715640/236651912-9bc75db2-307f-4efe-aa6c-fb4b472d0af6.png)|

  - What I think you can work on:
    - [ ] Ensure you use the `width` property on important containing elements of other contents (i.e. 100%)
    - [ ] Let's see how to fix the scrollbar showing up on the `x-axis` indicating an overflow.
    - [ ] There's an unnecessary extension of the height of your page towards the footer which is the white space you see. Kindly fix that too.
    - [ ] On screens less than `768px`,  having those white spaces on both sides while the page floats is not a good design, and it will most definitely affect user's perception or experience with your page. Kindly ensure that the width extends to both sides of the available screen size. :hammer_and_wrench: